client/galaxy/scripts/layout/menu.js

```diff
@@ -34,315 +42,301 @@ export function logoutClick() {
-                        url: "workflows/invocations",
-                        target: "__use_router__",
-                    },
-                ],
-            };
-            if (Galaxy.config.visualizations_visible) {
-                userTab.menu.push({
-                    title: _l("Visualizations"),
-                    url: "visualizations/list",
-                    target: "__use_router__",
-                });
-            }
-            if (Galaxy.config.interactivetools_enable) {
-                userTab.menu[userTab.menu.length - 1].divider = true;
-                userTab.menu.push({
-                    title: _l("Active InteractiveTools"),
-                    url: "interactivetool_entry_points/list",
-                    target: "__use_router__",
-                });
-            }
-        }
-        this.add(userTab);
-        return new $.Deferred().resolve().promise();
-    },
-});
+    //
+    // Help tab.
+    //
+    const helpTab = {
+        id: "help",
+        title: _l("Help"),
+        url: "javascript:void(0)",
+        tooltip: _l("Support, contact, and community"),
+        menu: [
+            {
+                title: _l("Support"),
+                url: options.support_url,
+                target: "_blank",
+            },
+            {
+                title: _l("Search"),
+                url: options.search_url,
+                target: "_blank",
+            },
+            {
+                title: _l("Mailing Lists"),
+                url: options.mailing_lists,
+                target: "_blank",
+            },
+            {
+                title: _l("Videos"),
+                url: options.screencasts_url,
+                target: "_blank",
+            },
+            {
+                title: _l("Wiki"),
+                url: options.wiki_url,
+                target: "_blank",
+            },
+            {
+                title: _l("How to Cite Galaxy"),
+                url: options.citation_url,
+                target: "_blank",
+            },
+            {
+                title: _l("Interactive Tours"),
+                url: "tours",
+            },
+        ],
+    };
+    if (options.terms_url) {
+        helpTab.menu.push({
+            title: _l("Terms and Conditions"),
+            url: options.terms_url,
+            target: "_blank",
+        });
+    }
+    if (options.helpsite_url) {
+        helpTab.menu.unshift({
+            title: _l("Galaxy Help"),
+            url: options.helpsite_url,
+            target: "_blank",
+        });
+    }
+    menu.push(helpTab);
+
+    // Load Webhook menu items, find better place to put this.
+    loadWebhookMenuItems();
 
-export default {
-    Collection: Collection,
-};
+    return menu;
+}
+
+function loadWebhookMenuItems() {
+    //
+    // Webhooks
+    //
+    Webhooks.load({
+        type: "masthead",
+        callback: function (webhooks) {
+            $(document).ready(() => {
```

Comment:
not a big deal and I guess wouldn't worth time, if it doesn't just work out of the box... But with ``` document.addEventListener('DOMContentLoaded', callback);``` we might be able to get rid of ```import $ from "jquery";``` in this file

Reply:
I will make this change, thanks!

Reply:
Change included in #9822 https://github.com/galaxyproject/galaxy/pull/9822/commits/7aa10bca2c14c2f59c40dc2f04424bc087a18e73.

Reply:
Great!